packages/components/src/InputFile/InputFile.tsx

```diff
@@ -122,6 +129,18 @@ interface InputFileProps {
   onUploadComplete?(file: FileUpload): void;
 }
 
+interface RequestParams extends Omit<UploadParams, "key"> {
+  /**
+   * The file being uploaded
+   */
+  file: File;
+
+  /**
+   *
+   */
```

Comment:
Did you mean to add a documentation here? also if we can explain why that's set as `any` that would be great! We try to avoid adding `any` as a type in Atlantis and if we have no choice, we explain why.